config/sanitizer/README.md

```diff
@@ -53,7 +53,7 @@ These obviously force the standard to be required, and also disables compiler-sp
 
 ## Sanitizer Builds [`sanitizers.cmake`](sanitizers.cmake)
 
-Sanitizers are tools that perform checks during a program’s runtime and returns issues, and as such, along with unit testing, code coverage and static analysis, is another tool to add to the programmers toolbox. And of course, like the previous tools, are tragically simple to add into any project using CMake, allowing any project and developer to quickly and easily use.
+Sanitizers are tools that perform checks during a program's runtime and returns issues, and as such, along with unit testing, code coverage and static analysis, is another tool to add to the programmers toolbox. And of course, like the previous tools, are tragically simple to add into any project using CMake, allowing any project and developer to quickly and easily use.
```

Comment:
This sentence has some subject/verb disagreements.  Should be tools that perform ... and **return** issues - and - Sanitizers ... **are** another tool to add ...  And maybe trivially simple instead of tragically simple, or just simple?

Reply:
updated.